### PR TITLE
[RFC] New package: unityhub-3.2.0

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by Nathan-MV** [comment]
It's working fine, when you have time can you update the PR to `3.2.0`?

**Comment by oreo639** [comment]
> It's working fine, when you have time can you update the PR to `3.2.0`?

Sure.

**Comment by kingpingu** [comment]
I see this PR is going to close soon, can we get reviewers assigned to this? It's a valuable package to add

File: srcpkgs/unityhub/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'unityhub'
+pkgname=unityhub
+version=3.2.0
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+hostmakedepends="rpmextract"
+depends="gtk+3 libnotify xdg-utils libuuid libsecret"
+short_desc="Unity instalation manager"
```
Comment (kingpingu) [inline srcpkgs/unityhub/template:9]: nit: typo on "instalation"

Comment (oreo639) [reply srcpkgs/unityhub/template]: thanks.

**Review by kingpingu — APPROVED** [review]

**Review by Earthcomputer — APPROVED** [review]

**Comment by Earthcomputer** [comment]
Bump

**Pull request closed without merging**